lib/views/backend/spree/admin/payments/source_forms/_paypal_braintree.html.erb

```diff
@@ -11,6 +11,6 @@
   </div>
 
   <div id="card_form<%= id %>" data-hook style="display: none;">
-    <%= render partial: "spree/shared/braintree_hosted_fields", locals: { id: id } %>
+    <%= render partial: "spree/shared/braintree_hosted_fields", locals: { payment_method: payment_method } %>
```

Comment:
Would be great if we can keep the partial working for who is rendering it in their application in other places. What if we keep accepting the old `id` key as well and deprecate it inside the view?

Reply:
Yeah, great idea! How does the new version look?